telemetry/telemetry.go

```diff
@@ -286,6 +286,18 @@ func CommandLine(command string, commandLine []string) {
 	)
 }
 
+// Report the platform and version of an attempted integration
+func InstallIntegrationVersion(integration, arch, platform, version string) {
+	analyticsClient.Track(distinctID(),
+		fmt.Sprintf("Install %s", integration),
+		map[string]any{
```

Comment:
probably doesn't matter much but won't this print in nondeterministic order? feels like a list might be a better choice

Reply:
Sorry, I either don't get your point or you seem to be treating the map as an argument to the print. But the map is the tags for the event created by Track(), and has to be a map for Segment/Mixpanel?

Reply:
ah my bad, yes I misparsed the map as an argument to the format string. LGTM :+1: